### PR TITLE
chore(runners): refactor test to avoid random failing tests

### PR DESCRIPTION
**Pull request by npalm**
Tests for scale down are failing random.

- Fixed test data to esnue scale down is always slecting the same runner to be terminated. This was causing the random failure.
- Cleanup the test file a bit, aka boys scout rule

### CODE REVIEW

File: lambdas/functions/control-plane/src/scale-runners/scale-down.test.ts [hunk 1]
```diff
@@ -86,7 +86,7 @@ let DEFAULT_ORG_RUNNERS_ORPHANED: RunnerInfo[];
 // i-running-111 | Repo | running and not exceeding minimumRunningTimeInMinutes
 // i-running-112 | Org | busy
 // i-running-113 | Repo | busy
-const oldest = moment(new Date()).subtract(25, 'minutes').toDate();
+const oldest = moment(new Date()).subtract(26, 'minutes').toDate();
```
Comment (PatrickMennen) [inline lambdas/functions/control-plane/src/scale-runners/scale-down.test.ts:89]: not related to this specific test set, but momentjs has been in "maintenance" for over a year. read more on https://momentjs.com/docs/#/-project-status/

File: lambdas/functions/control-plane/src/scale-runners/scale-down.test.ts [hunk 2]
```diff
@@ -391,35 +387,40 @@ describe('scaleDown', () => {
...
         timeZone: 'Europe/Amsterdam',
         evictionStrategy: 'oldest_first',
       };
+
       beforeEach(() => {
         process.env.SCALE_DOWN_CONFIG = JSON.stringify([defaultConfig]);
       });
 
-      it('Terminates 1 runner owned by orgs', async () => {
+      it('Should terminates based on the the idle config', async () => {
```
Comment (PatrickMennen) [inline lambdas/functions/control-plane/src/scale-runners/scale-down.test.ts:402]: terminate *

File: lambdas/functions/control-plane/src/scale-runners/scale-down.test.ts [hunk 3]
```diff
@@ -391,35 +387,40 @@ describe('scaleDown', () => {
...
+        // run test again with out idle config
+        jest.clearAllMocks();
+        mockListRunners.mockResolvedValue(RUNNERS_ORG_WITH_AUTO_SCALING_CONFIG);
+        await scaleDown();
+        expect(terminateRunner).toBeCalledTimes(2);
       });
 
-      it('Terminates 0 runners owned by org', async () => {
+      it('Should terminates 0 runners owned by org', async () => {
+        jest.clearAllMocks();
```
Comment (PatrickMennen) [inline lambdas/functions/control-plane/src/scale-runners/scale-down.test.ts:421]: I would probably move this to a beforeEach on your highest describe

**Review by PatrickMennen — CHANGES_REQUESTED** [review]

File: lambdas/functions/control-plane/src/scale-runners/scale-down.test.ts (hunk 1 above)
Comment (npalm) [reply lambdas/functions/control-plane/src/scale-runners/scale-down.test.ts:89]: Creeted #3435

File: lambdas/functions/control-plane/src/scale-runners/scale-down.test.ts (hunk 3 above)
Comment (npalm) [reply lambdas/functions/control-plane/src/scale-runners/scale-down.test.ts:421]: Left over from debugging, removed

**Review by PatrickMennen — APPROVED** [review]

**Pull request merged**